### PR TITLE
print view and progress bar updates

### PR DESCRIPTION
**Pull request by bethshook**

### CODE REVIEW

File: .ladle/config.mjs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export default {
   base: "/assessment-components/",
   outDir: "docs",
-  defaultStory: "card--default"
+  defaultStory: "print--default",
+  mode: "preview"
```
Comment (bethshook) [inline .ladle/config.mjs:5]: annoying that this is necessary for the playwright navigation to work. I tried updating ladle but that made no difference.

File: scripts/download-json.ts [hunk 2]
```diff
@@ -48,11 +48,11 @@ import fetch from 'node-fetch';
     console.log(`Progress: ${exercises.length}/${total_count}`);
   }
 
-  writeFile(filename, JSON.stringify(exercises), (err) => {
+  writeFile('exercises.json', JSON.stringify({title, exercises}), (err) => {
     if (err)
       console.error(err);
     else {
-      console.log(`${exercises.length} exercises written to ${filename}`);
+      console.log(`${exercises.length} exercises written to exercises.json`);
```
Comment (bethshook) [inline scripts/download-json.ts:55]: At the moment we always want the data saving to the same file. I also added a title arg which is used as a heading for the pdf.

File: .ladle/config.mjs (hunk 1 above)
Comment (jivey) [reply .ladle/config.mjs:5]: Yeah that was weird, and didn't seem like a caching bug either 🤷 Maybe something that gets fixed down the line

**Review by jivey — COMMENTED** [review]
ProgressBar needs an export in `src/index.ts` but other than that this looks good

**Review by jivey — APPROVED** [review]
Oops meant to approve, there may be some conflicts with the other pr but I think it'll be minimal

File: src/components/Exercise.tsx [hunk 3]
```diff
@@ -72,6 +74,7 @@ export const Exercise = ({
           canUpdateCurrentStep={canAnswer}
           displaySolution={true}
           detailedSolution={state.solution?.content_html}
+          show_all_feedback={show_all_feedback}
```
Comment (bethshook) [inline src/components/Exercise.tsx:77]: necessary to show answer feedback when there is no question-level `feedback_html` or `correct_answer_feedback_html`

**Review by jivey — APPROVED** [review]

**Pull request merged**